[04:41:22 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:root,passwd:123456,db:mytest,port:9066
[04:41:22 DBUtil.py L:32 INFO] execute sql:reload @@config_all in 172.100.9.1
[04:41:22 DBUtil.py L:37 INFO] sql executed without err
[04:41:22 DBUtil.py L:86 INFO] create query connection with host:172.100.9.1,user:testA,passwd:testA,port:8066
[04:41:22 DBUtil.py L:109 INFO] execute sql:show databases in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:22 DBUtil.py L:116 INFO] ('mytestA',)
[04:41:22 DBUtil.py L:109 INFO] execute sql:use mytestB in 172.100.9.1
[04:41:22 DBUtil.py L:112 INFO] host:172.100.9.1 execute sql err: (1044, "Access denied for user 'testA' to database 'mytestB'")
[04:41:22 DBUtil.py L:109 INFO] execute sql:drop table if exists mytestA.test2 in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:22 DBUtil.py L:109 INFO] execute sql:create table mytestA.test2(id int) in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:22 DBUtil.py L:109 INFO] execute sql:drop table if exists mytestA.test2 in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:22 DBUtil.py L:86 INFO] create query connection with host:172.100.9.1,user:testB,passwd:testB,port:8066
[04:41:22 DBUtil.py L:109 INFO] execute sql:show databases in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:22 DBUtil.py L:116 INFO] ('mytestB',)
[04:41:22 DBUtil.py L:109 INFO] execute sql:use mytestA in 172.100.9.1
[04:41:22 DBUtil.py L:112 INFO] host:172.100.9.1 execute sql err: (1044, "Access denied for user 'testB' to database 'mytestA'")
[04:41:22 DBUtil.py L:109 INFO] execute sql:drop table if exists mytestB.test1 in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:22 DBUtil.py L:109 INFO] execute sql:create table mytestB.test1(id int) in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:22 DBUtil.py L:109 INFO] execute sql:drop table if exists mytestB.test1 in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:22 DBUtil.py L:86 INFO] create query connection with host:172.100.9.1,user:testC,passwd:testC,port:8066
[04:41:22 DBUtil.py L:109 INFO] execute sql:show databases in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:22 DBUtil.py L:116 INFO] ('mytestC',)
[04:41:22 DBUtil.py L:109 INFO] execute sql:use mytestD in 172.100.9.1
[04:41:22 DBUtil.py L:112 INFO] host:172.100.9.1 execute sql err: (1044, "Access denied for user 'testC' to database 'mytestD'")
[04:41:22 DBUtil.py L:109 INFO] execute sql:drop table if exists mytestC.sbtestC1 in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:22 DBUtil.py L:109 INFO] execute sql:create table mytestC.sbtestC1(id int) in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:22 DBUtil.py L:109 INFO] execute sql:drop table if exists mytestC.sbtestC1 in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:22 DBUtil.py L:86 INFO] create query connection with host:172.100.9.1,user:testD,passwd:testD,port:8066
[04:41:22 DBUtil.py L:109 INFO] execute sql:show databases in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:22 DBUtil.py L:116 INFO] ('mytestD',)
[04:41:22 DBUtil.py L:109 INFO] execute sql:use mytestC in 172.100.9.1
[04:41:22 DBUtil.py L:112 INFO] host:172.100.9.1 execute sql err: (1044, "Access denied for user 'testD' to database 'mytestC'")
[04:41:22 DBUtil.py L:109 INFO] execute sql:drop table if exists mytestD.sbtestD1 in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:22 DBUtil.py L:109 INFO] execute sql:create table mytestD.sbtestD1(id int) in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:22 DBUtil.py L:109 INFO] execute sql:drop table if exists mytestD.sbtestD1 in 172.100.9.1
[04:41:22 DBUtil.py L:114 INFO] sql executed without err
[04:41:23 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:root,passwd:123456,db:mytest,port:9066
[04:41:23 DBUtil.py L:32 INFO] execute sql:reload @@config_all in 172.100.9.1
[04:41:23 DBUtil.py L:37 INFO] sql executed without err
[04:41:25 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:root,passwd:123456,db:mytest,port:9066
[04:41:25 DBUtil.py L:32 INFO] execute sql:reload @@config_all in 172.100.9.1
[04:41:25 DBUtil.py L:37 INFO] sql executed without err
[04:41:25 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:test,passwd:test,db:mytest,port:8066
[04:41:25 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:test_readonly,passwd:test_readonly,db:mytest,port:8066
[04:41:25 DBUtil.py L:32 INFO] execute sql:drop table if exists readonly_table in 172.100.9.1
[04:41:25 DBUtil.py L:37 INFO] sql executed without err
[04:41:25 DBUtil.py L:32 INFO] execute sql:create table readonly_table(id int, data varchar(10)) in 172.100.9.1
[04:41:25 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1495, 'User READ ONLY')
[04:41:25 DBUtil.py L:32 INFO] execute sql:create table readonly_table(id int, data varchar(10)) in 172.100.9.1
[04:41:25 DBUtil.py L:37 INFO] sql executed without err
[04:41:25 DBUtil.py L:32 INFO] execute sql:drop table readonly_table in 172.100.9.1
[04:41:25 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1495, 'User READ ONLY')
[04:41:25 DBUtil.py L:32 INFO] execute sql:alter table readonly_table add column data1 varchar(10) in 172.100.9.1
[04:41:25 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1495, 'User READ ONLY')
[04:41:25 DBUtil.py L:32 INFO] execute sql:insert into readonly_table values (1, 'aaa') in 172.100.9.1
[04:41:25 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1495, 'User READ ONLY')
[04:41:25 DBUtil.py L:32 INFO] execute sql:update readonly_table set data = 'bbb' where id = 1  in 172.100.9.1
[04:41:25 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1495, 'User READ ONLY')
[04:41:25 DBUtil.py L:32 INFO] execute sql:delete from readonly_table in 172.100.9.1
[04:41:25 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1495, 'User READ ONLY')
[04:41:25 DBUtil.py L:32 INFO] execute sql:select * from readonly_table in 172.100.9.1
[04:41:25 DBUtil.py L:37 INFO] sql executed without err
[04:41:25 DBUtil.py L:32 INFO] execute sql:drop table if exists readonly_table in 172.100.9.1
[04:41:25 DBUtil.py L:37 INFO] sql executed without err
[04:41:26 DBUtil.py L:32 INFO] execute sql:reload @@config_all in 172.100.9.1
[04:41:26 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:root,passwd:123456,db:mytest,port:9066
[04:41:27 DBUtil.py L:32 INFO] execute sql:reload @@config_all in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:test,passwd:test,db:mytest,port:8066
[04:41:27 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:testA,passwd:testA,db:mytest,port:8066
[04:41:27 DBUtil.py L:32 INFO] execute sql:drop table if exists schema_permission in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:create table schema_permission(id int, data varchar(10)) in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:delete from schema_permission where id = 1 in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:DELETE FROM schema_permission\nWHERE id = 1')
[04:41:27 DBUtil.py L:32 INFO] execute sql:select * from schema_permission in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT *\nFROM schema_permission')
[04:41:27 DBUtil.py L:32 INFO] execute sql:update schema_permission set data='AAA' where id = 1 in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:UPDATE schema_permission\nSET data = 'AAA'\nWHERE id = 1")
[04:41:27 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (5, 'eee') in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:INSERT INTO schema_permission\nVALUES (5, 'eee')")
[04:41:27 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:testB,passwd:testB,db:mytest,port:8066
[04:41:27 DBUtil.py L:32 INFO] execute sql:drop table if exists schema_permission in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:create table schema_permission(id int, data varchar(10)) in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:delete from schema_permission where id = 1 in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:select * from schema_permission in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:39 INFO] (2L, 'bbb')
[04:41:27 DBUtil.py L:39 INFO] (3L, 'ccc')
[04:41:27 DBUtil.py L:39 INFO] (4L, 'ddd')
[04:41:27 DBUtil.py L:32 INFO] execute sql:update schema_permission set data='AAA' where id = 1 in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (5, 'eee') in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:testC,passwd:testC,db:mytest,port:8066
[04:41:27 DBUtil.py L:32 INFO] execute sql:drop table if exists schema_permission in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:create table schema_permission(id int, data varchar(10)) in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:delete from schema_permission where id = 1 in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:select * from schema_permission in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT *\nFROM schema_permission')
[04:41:27 DBUtil.py L:32 INFO] execute sql:update schema_permission set data='AAA' where id = 1 in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:UPDATE schema_permission\nSET data = 'AAA'\nWHERE id = 1")
[04:41:27 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (5, 'eee') in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:INSERT INTO schema_permission\nVALUES (5, 'eee')")
[04:41:27 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:testD,passwd:testD,db:mytest,port:8066
[04:41:27 DBUtil.py L:32 INFO] execute sql:drop table if exists schema_permission in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:create table schema_permission(id int, data varchar(10)) in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:delete from schema_permission where id = 1 in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:DELETE FROM schema_permission\nWHERE id = 1')
[04:41:27 DBUtil.py L:32 INFO] execute sql:select * from schema_permission in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:39 INFO] (2L, 'bbb')
[04:41:27 DBUtil.py L:39 INFO] (3L, 'ccc')
[04:41:27 DBUtil.py L:39 INFO] (1L, 'aaa')
[04:41:27 DBUtil.py L:39 INFO] (4L, 'ddd')
[04:41:27 DBUtil.py L:32 INFO] execute sql:update schema_permission set data='AAA' where id = 1 in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:UPDATE schema_permission\nSET data = 'AAA'\nWHERE id = 1")
[04:41:27 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (5, 'eee') in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:INSERT INTO schema_permission\nVALUES (5, 'eee')")
[04:41:27 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:testE,passwd:testE,db:mytest,port:8066
[04:41:27 DBUtil.py L:32 INFO] execute sql:drop table if exists schema_permission in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:create table schema_permission(id int, data varchar(10)) in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:delete from schema_permission where id = 1 in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:DELETE FROM schema_permission\nWHERE id = 1')
[04:41:27 DBUtil.py L:32 INFO] execute sql:select * from schema_permission in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT *\nFROM schema_permission')
[04:41:27 DBUtil.py L:32 INFO] execute sql:update schema_permission set data='AAA' where id = 1 in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (5, 'eee') in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:INSERT INTO schema_permission\nVALUES (5, 'eee')")
[04:41:27 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:testF,passwd:testF,db:mytest,port:8066
[04:41:27 DBUtil.py L:32 INFO] execute sql:drop table if exists schema_permission in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:create table schema_permission(id int, data varchar(10)) in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:27 DBUtil.py L:32 INFO] execute sql:delete from schema_permission where id = 1 in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:DELETE FROM schema_permission\nWHERE id = 1')
[04:41:27 DBUtil.py L:32 INFO] execute sql:select * from schema_permission in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT *\nFROM schema_permission')
[04:41:27 DBUtil.py L:32 INFO] execute sql:update schema_permission set data='AAA' where id = 1 in 172.100.9.1
[04:41:27 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:UPDATE schema_permission\nSET data = 'AAA'\nWHERE id = 1")
[04:41:27 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (5, 'eee') in 172.100.9.1
[04:41:27 DBUtil.py L:37 INFO] sql executed without err
[04:41:28 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:root,passwd:123456,db:mytest,port:9066
[04:41:28 DBUtil.py L:32 INFO] execute sql:reload @@config_all in 172.100.9.1
[04:41:28 DBUtil.py L:37 INFO] sql executed without err
[04:41:28 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:root,passwd:123456,db:mytest,port:9066
[04:41:28 DBUtil.py L:32 INFO] execute sql:reload @@config_all in 172.100.9.1
[04:41:28 DBUtil.py L:37 INFO] sql executed without err
[04:41:29 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:root,passwd:123456,db:mytest,port:9066
[04:41:29 DBUtil.py L:32 INFO] execute sql:reload @@config_all in 172.100.9.1
[04:41:29 DBUtil.py L:37 INFO] sql executed without err
[04:41:29 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:test,passwd:test,db:mytest,port:8066
[04:41:29 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:readonly_schema1,passwd:readonly_schema1,db:mytest,port:8066
[04:41:29 DBUtil.py L:32 INFO] execute sql:drop table if exists schema_permission in 172.100.9.1
[04:41:29 DBUtil.py L:37 INFO] sql executed without err
[04:41:29 DBUtil.py L:32 INFO] execute sql:create table schema_permission(id int, data varchar(10)) in 172.100.9.1
[04:41:29 DBUtil.py L:37 INFO] sql executed without err
[04:41:29 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:29 DBUtil.py L:37 INFO] sql executed without err
[04:41:29 DBUtil.py L:32 INFO] execute sql:delete from schema_permission where id = 1 in 172.100.9.1
[04:41:29 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1495, 'User READ ONLY')
[04:41:29 DBUtil.py L:32 INFO] execute sql:select * from schema_permission in 172.100.9.1
[04:41:29 DBUtil.py L:37 INFO] sql executed without err
[04:41:29 DBUtil.py L:39 INFO] (1L, 'aaa')
[04:41:29 DBUtil.py L:39 INFO] (2L, 'bbb')
[04:41:29 DBUtil.py L:39 INFO] (4L, 'ddd')
[04:41:29 DBUtil.py L:39 INFO] (3L, 'ccc')
[04:41:29 DBUtil.py L:32 INFO] execute sql:update schema_permission set data='AAA' where id = 1 in 172.100.9.1
[04:41:29 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1495, 'User READ ONLY')
[04:41:29 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (5, 'eee') in 172.100.9.1
[04:41:29 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1495, 'User READ ONLY')
[04:41:29 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:readonly_schema2,passwd:readonly_schema2,db:mytest,port:8066
[04:41:29 DBUtil.py L:32 INFO] execute sql:drop table if exists schema_permission in 172.100.9.1
[04:41:29 DBUtil.py L:37 INFO] sql executed without err
[04:41:29 DBUtil.py L:32 INFO] execute sql:create table schema_permission(id int, data varchar(10)) in 172.100.9.1
[04:41:29 DBUtil.py L:37 INFO] sql executed without err
[04:41:29 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:29 DBUtil.py L:37 INFO] sql executed without err
[04:41:29 DBUtil.py L:32 INFO] execute sql:select * from schema_permission in 172.100.9.1
[04:41:29 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT *\nFROM schema_permission')
[04:41:29 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:readonly_schema3,passwd:readonly_schema3,db:mytest,port:8066
[04:41:29 DBUtil.py L:32 INFO] execute sql:drop table if exists schema_permission in 172.100.9.1
[04:41:29 DBUtil.py L:37 INFO] sql executed without err
[04:41:29 DBUtil.py L:32 INFO] execute sql:create table schema_permission(id int, data varchar(10)) in 172.100.9.1
[04:41:29 DBUtil.py L:37 INFO] sql executed without err
[04:41:29 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:29 DBUtil.py L:37 INFO] sql executed without err
[04:41:29 DBUtil.py L:32 INFO] execute sql:delete from schema_permission where id = 1 in 172.100.9.1
[04:41:29 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1495, 'User READ ONLY')
[04:41:29 DBUtil.py L:32 INFO] execute sql:select * from schema_permission in 172.100.9.1
[04:41:29 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT *\nFROM schema_permission')
[04:41:29 DBUtil.py L:32 INFO] execute sql:update schema_permission set data='AAA' where id = 1 in 172.100.9.1
[04:41:29 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1495, 'User READ ONLY')
[04:41:29 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (5, 'eee') in 172.100.9.1
[04:41:29 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1495, 'User READ ONLY')
[04:41:29 DBUtil.py L:32 INFO] execute sql:reload @@config_all in 172.100.9.1
[04:41:29 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:root,passwd:123456,db:mytest,port:9066
[04:41:30 DBUtil.py L:32 INFO] execute sql:reload @@config_all in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:test,passwd:test,db:mytest,port:8066
[04:41:30 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:schema_table1,passwd:schema_table1,db:mytest,port:8066
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists schema_permission in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:create table schema_permission(id int, data varchar(10)) in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:delete from schema_permission where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:select * from schema_permission in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:39 INFO] (3L, 'ccc')
[04:41:30 DBUtil.py L:39 INFO] (4L, 'ddd')
[04:41:30 DBUtil.py L:39 INFO] (2L, 'bbb')
[04:41:30 DBUtil.py L:32 INFO] execute sql:update schema_permission set data='AAA' where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (5, 'eee') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:create table table1(id int, data varchar(10)) in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table1 values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:delete from table1 where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:select * from table1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:39 INFO] (2L, 'bbb')
[04:41:30 DBUtil.py L:39 INFO] (3L, 'ccc')
[04:41:30 DBUtil.py L:39 INFO] (4L, 'ddd')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select a.*,b.* from schema_permission a,table1 b where a.id = b.id in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:39 INFO] (2L, 'bbb', 2L, 'bbb')
[04:41:30 DBUtil.py L:39 INFO] (3L, 'ccc', 3L, 'ccc')
[04:41:30 DBUtil.py L:39 INFO] (4L, 'ddd', 4L, 'ddd')
[04:41:30 DBUtil.py L:32 INFO] execute sql:update table1 set data='AAA' where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table1 values (5, 'eee') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table2 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:create table table2(id int, data varchar(10)) in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table2 values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:delete from table2 where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:DELETE FROM table2\nWHERE id = 1')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select * from table2 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT *\nFROM table2')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select a.*,b.* from schema_permission a,table2 b where a.id = b.id in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT a.*, b.*\nFROM schema_permission a, table2 b\nWHERE a.id = b.id')
[04:41:30 DBUtil.py L:32 INFO] execute sql:update table2 set data='AAA' where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:UPDATE table2\nSET data = 'AAA'\nWHERE id = 1")
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table2 values (5, 'eee') in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:INSERT INTO table2\nVALUES (5, 'eee')")
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table2 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table3 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:create table table3(id int, data varchar(10)) in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table3 values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:delete from table3 where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:select * from table3 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT *\nFROM table3')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select a.*,b.* from schema_permission a,table3 b where a.id = b.id in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT a.*, b.*\nFROM schema_permission a, table3 b\nWHERE a.id = b.id')
[04:41:30 DBUtil.py L:32 INFO] execute sql:update table3 set data='AAA' where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:UPDATE table3\nSET data = 'AAA'\nWHERE id = 1")
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table3 values (5, 'eee') in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:INSERT INTO table3\nVALUES (5, 'eee')")
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table3 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table4 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:create table table4(id int, data varchar(10)) in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table4 values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:delete from table4 where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:DELETE FROM table4\nWHERE id = 1')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select * from table4 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:39 INFO] (1L, 'aaa')
[04:41:30 DBUtil.py L:39 INFO] (2L, 'bbb')
[04:41:30 DBUtil.py L:39 INFO] (3L, 'ccc')
[04:41:30 DBUtil.py L:39 INFO] (4L, 'ddd')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select a.*,b.* from schema_permission a,table4 b where a.id = b.id in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:39 INFO] (2L, 'bbb', 2L, 'bbb')
[04:41:30 DBUtil.py L:39 INFO] (3L, 'ccc', 3L, 'ccc')
[04:41:30 DBUtil.py L:39 INFO] (4L, 'ddd', 4L, 'ddd')
[04:41:30 DBUtil.py L:32 INFO] execute sql:update table4 set data='AAA' where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:UPDATE table4\nSET data = 'AAA'\nWHERE id = 1")
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table4 values (5, 'eee') in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:INSERT INTO table4\nVALUES (5, 'eee')")
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table4 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table5 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:create table table5(id int, data varchar(10)) in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table5 values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:delete from table5 where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:DELETE FROM table5\nWHERE id = 1')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select * from table5 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT *\nFROM table5')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select a.*,b.* from schema_permission a,table5 b where a.id = b.id in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT a.*, b.*\nFROM schema_permission a, table5 b\nWHERE a.id = b.id')
[04:41:30 DBUtil.py L:32 INFO] execute sql:update table5 set data='AAA' where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table5 values (5, 'eee') in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:INSERT INTO table5\nVALUES (5, 'eee')")
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table5 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table6 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:create table table6(id int, data varchar(10)) in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table6 values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:delete from table6 where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:DELETE FROM table6\nWHERE id = 1')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select * from table6 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT *\nFROM table6')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select a.*,b.* from schema_permission a,table6 b where a.id = b.id in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT a.*, b.*\nFROM schema_permission a, table6 b\nWHERE a.id = b.id')
[04:41:30 DBUtil.py L:32 INFO] execute sql:update table6 set data='AAA' where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:UPDATE table6\nSET data = 'AAA'\nWHERE id = 1")
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table6 values (5, 'eee') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table6 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:11 INFO] create query connection with host:172.100.9.1,user:schema_table2,passwd:schema_table2,db:mytest,port:8066
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists schema_permission in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:create table schema_permission(id int, data varchar(10)) in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:delete from schema_permission where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:DELETE FROM schema_permission\nWHERE id = 1')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select * from schema_permission in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT *\nFROM schema_permission')
[04:41:30 DBUtil.py L:32 INFO] execute sql:update schema_permission set data='AAA' where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:UPDATE schema_permission\nSET data = 'AAA'\nWHERE id = 1")
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into schema_permission values (5, 'eee') in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:INSERT INTO schema_permission\nVALUES (5, 'eee')")
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:create table table1(id int, data varchar(10)) in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table1 values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:delete from table1 where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:select * from table1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:39 INFO] (2L, 'bbb')
[04:41:30 DBUtil.py L:39 INFO] (3L, 'ccc')
[04:41:30 DBUtil.py L:39 INFO] (4L, 'ddd')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select a.*,b.* from schema_permission a,table1 b where a.id = b.id in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT a.*, b.*\nFROM schema_permission a, table1 b\nWHERE a.id = b.id')
[04:41:30 DBUtil.py L:32 INFO] execute sql:update table1 set data='AAA' where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table1 values (5, 'eee') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table2 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:create table table2(id int, data varchar(10)) in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table2 values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:delete from table2 where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:DELETE FROM table2\nWHERE id = 1')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select * from table2 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT *\nFROM table2')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select a.*,b.* from schema_permission a,table2 b where a.id = b.id in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT a.*, b.*\nFROM schema_permission a, table2 b\nWHERE a.id = b.id')
[04:41:30 DBUtil.py L:32 INFO] execute sql:update table2 set data='AAA' where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:UPDATE table2\nSET data = 'AAA'\nWHERE id = 1")
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table2 values (5, 'eee') in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:INSERT INTO table2\nVALUES (5, 'eee')")
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table2 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table3 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:create table table3(id int, data varchar(10)) in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table3 values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:delete from table3 where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:select * from table3 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT *\nFROM table3')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select a.*,b.* from schema_permission a,table3 b where a.id = b.id in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT a.*, b.*\nFROM schema_permission a, table3 b\nWHERE a.id = b.id')
[04:41:30 DBUtil.py L:32 INFO] execute sql:update table3 set data='AAA' where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:UPDATE table3\nSET data = 'AAA'\nWHERE id = 1")
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table3 values (5, 'eee') in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:INSERT INTO table3\nVALUES (5, 'eee')")
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table3 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table4 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:create table table4(id int, data varchar(10)) in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table4 values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:delete from table4 where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:DELETE FROM table4\nWHERE id = 1')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select * from table4 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:39 INFO] (1L, 'aaa')
[04:41:30 DBUtil.py L:39 INFO] (2L, 'bbb')
[04:41:30 DBUtil.py L:39 INFO] (3L, 'ccc')
[04:41:30 DBUtil.py L:39 INFO] (4L, 'ddd')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select a.*,b.* from schema_permission a,table4 b where a.id = b.id in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT a.*, b.*\nFROM schema_permission a, table4 b\nWHERE a.id = b.id')
[04:41:30 DBUtil.py L:32 INFO] execute sql:update table4 set data='AAA' where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:UPDATE table4\nSET data = 'AAA'\nWHERE id = 1")
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table4 values (5, 'eee') in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:INSERT INTO table4\nVALUES (5, 'eee')")
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table4 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table5 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:create table table5(id int, data varchar(10)) in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table5 values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:delete from table5 where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:DELETE FROM table5\nWHERE id = 1')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select * from table5 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT *\nFROM table5')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select a.*,b.* from schema_permission a,table5 b where a.id = b.id in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT a.*, b.*\nFROM schema_permission a, table5 b\nWHERE a.id = b.id')
[04:41:30 DBUtil.py L:32 INFO] execute sql:update table5 set data='AAA' where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table5 values (5, 'eee') in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:INSERT INTO table5\nVALUES (5, 'eee')")
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table5 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table6 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:create table table6(id int, data varchar(10)) in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table6 values (1,'aaa'),(2,'bbb'),(3,'ccc'),(4,'ddd') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:delete from table6 where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:DELETE FROM table6\nWHERE id = 1')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select * from table6 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT *\nFROM table6')
[04:41:30 DBUtil.py L:32 INFO] execute sql:select a.*,b.* from schema_permission a,table6 b where a.id = b.id in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, 'The statement DML privilege check is not passed, sql:SELECT a.*, b.*\nFROM schema_permission a, table6 b\nWHERE a.id = b.id')
[04:41:30 DBUtil.py L:32 INFO] execute sql:update table6 set data='AAA' where id = 1 in 172.100.9.1
[04:41:30 DBUtil.py L:35 INFO] host:172.100.9.1 execute sql err: (1064, "The statement DML privilege check is not passed, sql:UPDATE table6\nSET data = 'AAA'\nWHERE id = 1")
[04:41:30 DBUtil.py L:32 INFO] execute sql:insert into table6 values (5, 'eee') in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:drop table if exists table6 in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
[04:41:30 DBUtil.py L:32 INFO] execute sql:reload @@config_all in 172.100.9.1
[04:41:30 DBUtil.py L:37 INFO] sql executed without err
